.github/config.yml

```diff
@@ -19,7 +19,8 @@ firstPRMergeComment: >
    
   ![Congrats!](https://raw.githubusercontent.com/layer5io/layer5/master/.github/welcome/Layer5-celebration.png)
 
-  Please [star the project](../stargazers) if you have yet to do so.
+   &nbsp;
```

Comment:
Hi @Josephchinedu I don’t think there should be any space in line 22 [see this](https://github.com/layer5io/service-mesh-performance/blob/master/.github/config.yml)